tests/providers/amazon/aws/sensors/test_batch.py

```diff
@@ -44,45 +42,38 @@ def setUp(self):
     @mock.patch.object(BatchClientHook, "get_job_description")
     def test_poke_on_success_state(self, mock_get_job_description):
         mock_get_job_description.return_value = {"status": "SUCCEEDED"}
-        self.assertTrue(self.batch_sensor.poke({}))
+        assert self.batch_sensor.poke({})
```

Comment:
Non-blocking suggestion:  missed one `is True` here

Reply:
Ahhh dang.
I included this change here: https://github.com/apache/airflow/pull/28145/commits/9960d4325c4f98e807b4019625a566cfb26db18c

Reply:
Jarek sniped me on the merge, but it's all good.  It's a minor thing.